pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -684,12 +687,12 @@ func TestZeroRequest(t *testing.T) {
 			}
 			for _, hp := range list {
 				if test.name == "test priority of larger pod with machine with zero-request pod" {
```

Comment:
We actually want to get rid of this `if` statement here. We should not rely on the name of the test. Please remove this if statement and provide the expected score in the corresponding test case.

Reply:
@bsalamat Sorry, I thougt the score of the third case varies.